src/lib.rs

```diff
@@ -14,3 +14,6 @@ pub mod config;
 
 /// Crate-wide utility macros
 pub mod macros;
+
+/// Utility for Writing to Scoring Report
```

Comment:
`writing`, `scoring`, `report`

+ we'll probably abstract this to a `scoring` folder where we'll put remote too, so you should do something like

```
src
+- scoring
   +- mod.rs
   +- report.rs